db/migrate/20230220233732_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+
```

Comment:
- [ ] Great job for adding the columns here in users.rb but I think you forget to add is user table `posts_counter` as shown in the ERD diagram so kindly add a new migration and add this column 🤩💯⭐✨

### You can do somthing like this ⬇💪🏻 This will add a new migration for you!
> rails generate migration AddPostsCounterToUsers

### And now inside your migration add a new column 💯💯⭐
> add_column :users, :posts_counter, :integer